scm-ui/ui-components/src/layout/Footer.tsx

```diff
@@ -45,7 +46,12 @@ type TitleWithIconsProps = {
 
 const TitleWithIcon: FC<TitleWithIconsProps> = ({ icon, title }) => (
   <>
-    <i className={`fas fa-${icon} fa-fw`} /> {title}
+    <i
+      className={`fas fa-${icon} fa-fw`}
+      //@ts-ignore replace the spaces with dashes since cypress won't find it otherwise
```

Comment:
Why the `@ts-ignore`?

Reply:
I don't see any reason for it

Reply:
TS2339: Property 'replaceAll' does not exist on type 'string'.

Reply:
Okay, then it's maybe not a good idea to just ignore that.

https://caniuse.com/#feat=mdn-javascript_builtins_string_replaceall